client/layout/transient-notices/index.js

```diff
@@ -27,6 +27,7 @@ class TransientNotices extends Component {
 				notices={ notices }
 				className={ classes }
 				onRemove={ onRemove }
+				onRemove2={ onRemove2 }
```

Comment:
I pushed a commit to add this -- otherwise, the new notices that we are using in CES do not get removed. :-)

Reply:
Oh whoops! I must have accidentally removed that when cleaning up test code. Thanks!